ui/app/models/pki/certificate/generate.js

```diff
@@ -5,21 +5,20 @@ import PkiCertificateBaseModel from './base';
 
 const generateFromRole = [
   {
-    default: ['commonName'],
+    default: ['commonName', 'customTtl'],
```

Comment:
leveraging joint component (also used in generating root)
<img width="553" alt="Screenshot 2023-02-09 at 5 16 07 PM" src="https://user-images.githubusercontent.com/68122737/217975661-ca7caa72-569b-4566-8ce9-4ebe1b460cf8.png">